### DISCUSSION TITLE
如何添加多个节点到xray服务器

### DISCUSSION THREAD

**Discussion by PunkFleet** [XTLS/Xray-core#2994] [Q&A] [answered]
我已经实现了一个xray服务器。但是我想要用另一台其他地区的服务器作为备用节点。
我知道我可以再用其他的子域名搭建第二台服务器来实现。但是。。。有另一个方法吗？一个子域名使用两个服务器，然后在客户端能够自主切换。

**Comment by qwerr0** [comment]
一个域名带有多个A记录，那么应该怎么区分哪个是主要使用的节点的记录和哪个是备用节点的记录呢？
比如主要服务器用 main.proxy.example.com, 备用服务器用 fallback.proxy.example.com 就很清晰明了，再配置成 Vless 的 VNext 就很 OK

**Comment by PunkFleet** [comment]
> 比如主要服务器用 main.proxy.example.com, 备用服务器用 fallback.proxy.example.com 就很清晰明了，再配置成 Vless 的 VNext 就很 OK

😢没理解. VNext 是什么？ 在官方配置文档中没有发现有这个名词

**Reply by qwerr0** [reply, marked as answer]
outbounds vless 的 settings 里的结构是:
```json
      "settings": {
        "vnext": [ // < ---- 这个
          {
            "address": "服务器地址",
            "port": 端口,
            "users": [
              {
                "id": "UUID",
                "flow": "xtls-rprx-vision",
                "encryption": "none"
              }
            ]
          },
// 继续添加 多个 服务器
        ]
      },
```
